onnxruntime/core/framework/path_lib.h

```diff
@@ -20,6 +20,17 @@
 #include "core/session/onnxruntime_c_api.h"
 
 using PATH_CHAR_TYPE = ORTCHAR_T;
```

Comment:
>ORTCHAR_T [](start = 23, length = 9)

I can not read this time so that means others would be able to either.

Reply:
Change canceled.